src/screens/MediaEpisode/MediaEpisode.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { useNavigate } from 'react-router';
+import { Helmet } from 'react-helmet';
+import { useTranslation } from 'react-i18next';
+import shallow from 'zustand/shallow';
+
+import styles from './MediaSeries.module.scss';
+
+import useEntitlement from '#src/hooks/useEntitlement';
+import CardGrid from '#src/components/CardGrid/CardGrid';
+import useBlurImageUpdater from '#src/hooks/useBlurImageUpdater';
+import { episodeURL, formatSeriesMetaString, formatVideoMetaString } from '#src/utils/formatting';
+import Filter from '#src/components/Filter/Filter';
+import VideoDetails from '#src/components/VideoDetails/VideoDetails';
+import useMedia from '#src/hooks/useMedia';
+import { useSeriesData } from '#src/hooks/useSeriesData';
+import ErrorPage from '#src/components/ErrorPage/ErrorPage';
+import { generateEpisodeJSONLD } from '#src/utils/structuredData';
+import { enrichMediaItems, filterSeries, getFiltersFromSeries, getNextItemId } from '#src/utils/series';
+import LoadingOverlay from '#src/components/LoadingOverlay/LoadingOverlay';
+import { useWatchHistoryStore } from '#src/stores/WatchHistoryStore';
+import { useConfigStore } from '#src/stores/ConfigStore';
+import { useAccountStore } from '#src/stores/AccountStore';
+import StartWatchingButton from '#src/containers/StartWatchingButton/StartWatchingButton';
+import useBreakpoint, { Breakpoint } from '#src/hooks/useBreakpoint';
+import Cinema from '#src/containers/Cinema/Cinema';
+import TrailerModal from '#src/containers/TrailerModal/TrailerModal';
+import ShareButton from '#src/components/ShareButton/ShareButton';
+import FavoriteButton from '#src/containers/FavoriteButton/FavoriteButton';
+import Button from '#src/components/Button/Button';
+import PlayTrailer from '#src/icons/PlayTrailer';
+import type { PlaylistItem } from '#types/playlist';
+import type { MediaScreen } from '#types/screens';
+import useQueryParam from '#src/hooks/useQueryParam';
+import useGetSeriesId from '#src/hooks/useGetSeriesId';
+
+const MediaEpisode: MediaScreen = ({ item }) => {
```

Comment:
Might be good to add "series" here (i.e. "MediaSeriesEpisode"), in case the meaning of episode isn't clear.

Reply:
Sure, no problem!